scribble-doc/scribblings/scribble/config.scrbl

```diff
@@ -640,6 +640,20 @@ style:
 
  ]
 
+Additionally, the @filepath{racket.tex} Latex configuration
```

Comment:
I have no opinion one way or the other. But the rest of the docs say Latex. So if we do make it LaTeX, we should change that consistently.